articles/user-guide/libraries/standard/diagnostics.md

```diff
@@ -88,12 +89,12 @@ That is,
 \begin{align}
     \ket{\psi} = \ket{\psi'} \text{ if and only if } \braket{\psi | \psi'} = 1.
 \end{align}
-Using the primitive operations defined in the prelude, we can directly perform a measurement that returns `Zero` if $\ket{\psi}$ is an eigenstate of one of the Pauli operators.
+Using the primitive operations defined in the prelude, you can directly perform a measurement that returns `Zero` if $\ket{\psi}$ is an eigenstate of one of the Pauli operators.
```

Comment:
This looks like another place where we can update language; "primitive" → "intrinsic", for example, and getting rid of "prelude."